mesh/tests/profiles_tests.py

```diff
@@ -89,6 +89,21 @@ def test_no_account_preferred_pronouns(self):
         self.assertEquals(json_response.get("status"), "error")
         self.assertEquals(json_response.get("message"), "An account does not exist with this account ID.")
 
+    def test_post_user_name (self):
+        user_name_data = {"userName": "kwame brown"}
+        response = self.client.post(f'/profiles/user-name/{self.test_profile.accountID}', user_name_data)
+        self.assertEquals(response.status_code, 200)
+
+    def test_post_preferred_name (self):
+        preferred_name_data = {"preferredName": "brown"}
+        response = self.client.post(f'/profiles/preferred-name/{self.test_profile.accountID}', preferred_name_data)
+        self.assertEquals(response.status_code, 200)
+
+    def test_post__preferred_pronouns (self):
```

Comment:
Pretty sure you didn't mean to put two underscores here

Reply:
👍